conbench/app/_plots.py

```diff
@@ -399,17 +415,26 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
     unit = history[0]["unit"]
     with_dist = [h for h in history if h["distribution_mean"]]
     formatted, axis_unit = _should_format(history, unit)
+    dist_change_ixs = [
+        ix
+        for ix, result in enumerate(history)
+        if result["change_annotations"].get("begins_distribution_change", False)
```

Comment:
is the `False` default important here? `d.get('unknown-key')` returns `None`.

Reply:
Nope, it's not important. None is falsey enough. 🙂 

I think this idiom was just copy/pasted from here, where it is important to be an actual boolean, because we're POSTing that value to the API.
https://github.com/conbench/conbench/blob/2b1653f5582bf397ffaa10278062949a89a2ff8b/conbench/app/benchmarks.py#L198-L200